.changeset/fresh-badgers-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+nginx proxy pass dev mode
```

Comment:
This needs a full description of what this PR does, phrased as if it's prefixed with "This PR..."